### DISCUSSION TITLE
guides/ci-cd/jenkins-ornatish

### DISCUSSION THREAD

**Discussion by giscus** [ismoilovdevml/devops-journey#26] [General]
# guides/ci-cd/jenkins-ornatish

Linux Serverlarga Jenkins o'rnatish

https://devops-journey.uz/guides/ci-cd/jenkins-ornatish

**Comment by aeshakhzod** [comment]
Salom bolalar!

**Reply by Pulatovusmonjon1** [reply]
Tashlab qo'ygan silkaga kirmayapdi aka

**Reply by ismoilovdevml** [reply]
> Tashlab qo'ygan silkaga kirmayapdi aka 

Qaysi link?
muammoni aniq aytingda
